payu/experiment.py

```diff
@@ -385,8 +397,7 @@ def setup(self, force_archive=False):
         self.manifest.make_links()
 
         # Copy manifests to work directory so they archived on completion
-        manifest_path = os.path.join(self.work_path, 'manifests')
-        self.manifest.copy_manifests(manifest_path)
+        self.manifest.copy_manifests(os.path.join(self.work_path, 'manifests'))
```

Comment:
Use two lines here?
```
manifest_path = os.path.join(self.work_path), 'manifests')
self.manifest.copy_manifests(manifest_path)
```
Or, maybe just add `self.manifest.dir` and set it during init?

Reply:
haha, just noticed this was previously two lines :D

Reply:
So change it back? Or set `self.manifest.dir`?

Reply:
Yeah lets revert back?